examples/summarization/train.py

```diff
@@ -97,29 +103,36 @@ def collate_fun(self, batch):
             max_length=self.max_length_decoder,
         )
         labels = target_tokens["input_ids"]
-        labels[labels[:, :] == self.model.config.pad_token_id] = -100
+        labels[labels[:, :] == self.pad_token_id] = -100
         source_tokens["labels"] = labels
         return source_tokens
+    
+    @property
+    def pad_token_id(self):
+        return self.model.config.pad_token_id
 
     def train_step(self, global_step: int, batch, device):
         batch = batch.to(device)
         result = self.model.forward(**batch)
         global_stats.update("lr", self.schedulers.get_last_lr()[0], frequent=True)
         return result.loss
+    
+    def get_core_model(self):
```

Comment:
can this be avoided? Looks like a redundant refatorization for deepspeed

Reply:
maybe - but i mostly did this so I don't have to copy paste the entire val_step again and instead just override get_core_model() which will vary for ORT/DS extension. I could do some fancy self.model property magic in ORT/Deepspeed extension but its just over kill for just wanting to get access to the original pytorch model after wrapping it twice for ORT/DS

Reply:
leaving it in because it's a very small overridable function that's much better than copy-pasting the entire val_step across extended modules.